Tasks/AzureResourceGroupDeployment/operations/MachineGroupExtensionHelper.ts

```diff
@@ -81,7 +81,7 @@ export class MachineGroupExtensionHelper {
             await this.addExtensionOnRunningVm(vm);
         }
         else {
-            throw (tl.loc("VMTransitioningSkipExtensionAddition", vmName));
+            throw new Error(tl.loc("VMTransitioningSkipExtensionAddition", vmName));
```

Comment:
In the function WriteDeploymentErrors, it expects an object with a message instead of a string